DCP_Tutorial.md

```diff
@@ -0,0 +1,135 @@
+## DCP Demo (self-led)
+**Target Audience:** DCP Staff
+
+**Purpose:** Serve as a learning resource for end-to-end data flow in the DCP
+
+**Components:** Ingest, Upload, Data store, Data pipelines/Analysis, Azul, Data Portal, Matrix
+
+**Reference Document:** https://docs.google.com/document/d/1-9k9tcuIc4aTuxIo-KAT34R_OSca0j9_3LDNJwqksBQ/edit#heading=h.kuwifzq1d62x.
+
+**Note:** This document was published in June 2018 so may have inconsistencies and steps missing. A v2 document will be published soon and linked here.)
+
+### FAQ
+**Who do I contact if I encounter any any issues while following the steps?**
+Feel free reach out to #dcp-ops-help, create an issue in the DCP repository, or ping Parth.
+
+**Do I need to have a certain level of technical proficiency to follow the steps?**
+Technical proficiency should not be a requirement. Please reach out if you encounter any issues, as this indicates that the instructions need additions or modifications.
+
+**How can I make edits to this document?**
+Please create a PR in the DCP repository.
+
+**Can I run this in production?**
+No. Please follow the instructions and only utilize the staging environment.
+
+**Will this project be available at tracker.staging.data.humancellatlas.org?**
+No. The tracker ignores all test projects in order to not overload the view.
+
+**What ids do I need to keep track of?**
+Submission ID, Project UUID, Primary Bundle UUID, Primary Bundle FQID, Workflow UUID, Analysis Bundle UUID, Analysis Bundle FQID. These will be defined as a part of the demo.
+
+### Demo
+**Requirements**
+```
+    # python version >= 3.6
+
+    # Run the command in terminal app of your choice
+    pip install hca --upgrade
+    open ~/.config/hca/config.json
+    # In the config file, update "swagger_url" to point to "https://dss.staging.data.humancellatlas.org/v1/swagger.json"
+
+    # Access to job manager
+    # https://job-manager.caas-prod.broadinstitute.org/jobs
+    # Contact #dcp-ops-help if you do not see any workflows in this view.
+
+    # Access to ingest UI
+    # https://ui.ingest.staging.data.humancellatlas.org
+    # Contact #dcp-ops-help if you receive a permission error when logging in.
+```
+#### Phase 1: Primary submission preparation
+**Components:** Ingest, Upload  
+**Description:** In this phase, you will be uploading a spreadsheet through the ingest UI in order to create a submission. Then, you will utilize the HCA cli to upload the data files for the submission. Both ingest and upload are currently running in AWS infrastructure.
+1) Please download a spreadsheet found at https://github.com/HumanCellAtlas/metadata-schema/tree/master/infrastructure_testing_files/current. These are test spreadsheets that are run through the DCP wide integration tests. Feel free to choose any of the options (Currently smart-seq-2/ss2 and 10x/optimus spreadsheets are available).
```

Comment:
Since the tutorial is using the staging environment, I think we want to use the test spreadsheet from the staging branch: https://github.com/HumanCellAtlas/metadata-schema/tree/staging/infrastructure_testing_files/current

Reply:
Good catch!